packages/main/src/TreeItem.ts

```diff
@@ -81,17 +39,48 @@ const metadata = {
  * @implements sap.ui.webc.main.ITreeItem
  * @since 1.0.0-rc.8
  */
+@customElement("ui5-tree-item")
 class TreeItem extends TreeItemBase {
+	/**
+	 * Defines the text of the tree item.
+	 *
+	 * @public
+	 * @type {string}
+	 * @name sap.ui.webc.main.TreeItem.prototype.text
+	 * @defaultValue ""
+	 */
+	@property()
+	text!: string;
+
+	/**
+	 * Defines the <code>additionalText</code>, displayed in the end of the tree item.
+	 * @type {string}
+	 * @name sap.ui.webc.main.TreeItem.prototype.additionalText
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property()
+	additionalText?: string;
```

Comment:
type string always have default value "" if not explicitly set to undefined. also add @defaultValue "" at jsdoc comment

Reply:
done